app/javascript/src/modules/organisations.js

```diff
@@ -22,63 +18,59 @@ async function searchResults (host, searchTerm) {
     }
   })
   const data = response.data.data
-  return updateMatchCounters(data, searchTerm)
+  return data
 }
 
-function setMatchAndCount (data = [], searchTerm = null) {
-  noMatchCount = 0
-  organisationMatches = data
-  previousSearchTerm = searchTerm
-}
+function showResults (results, inputText) {
+  if (results.length > 0) {
+    let organisationsContainer = document.querySelector('.govuk-radios') // with MP flag on
```

Comment:
i think MP this relates to multiple proceedings (code was originally copied from proceeding.js) and can be removed/simplified

Reply:
remove as this was leftover from "multiple proceedings" (MP flag) works done on the proceedings search page and javascript file, that is not required here.